packages/core/phase1/triggers/index.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-import { TriggerCode } from "../../types/TriggerCode"
+import TriggerCode from "bichard7-next-data-latest/dist/types/TriggerCode"
 import type { TriggerGenerator } from "../types/TriggerGenerator"
 
 const modules = Object.keys(TriggerCode)
-  .filter((code) => code !== TriggerCode.TRPR0028)
+  .filter((code) => code.startsWith("TRPR") && code !== TriggerCode.TRPR0028)
```

Comment:
We can refactor `code.startsWith("TRPR")` when we are working on merging post-update trigger code with pre-update trigger code.